standards/SPIFFE_Federation.md

```diff
@@ -0,0 +1,316 @@
+
+Bundle endpoint servers MAY respond with HTTP redirect (as defined by [RFC 7231 section 6.4][8]).  Bundle endpoint clients SHOULD follow the redirect if the URL meets all the requirements of a valid bundle endpoint URL.  When connecting to the new URL, the same TLS considerations MUST be applied as connecting to the original URL. In particular, it MUST present a valid X509-SVID for the same SPIFFE ID as originally configured.  Bundle endpoint clients SHOULD use the configured endpoint URL for each bundle refresh and SHOULD NOT permanently store the location for future fetches. See the [Security Considerations](#7-security-considerations) for more information. 
+
+## 6. Relationship Lifecycle
+This section describes the lifecycle of a federation "relationship", including the establishment of the first connection, ongoing maintenance, and termination.
+
+Federation relationships are one-way. In other words, Alice could have a relationship with Bob but not vice versa. In this case, Alice would be able to validate identities issued by Bob, but Bob would not know how to validate an identity issued by Alice.
+
+To achieve mutual authentication, two relationships are formed - one in each direction.
+
+### 6.1. Establishing a Relationship
+As described in the Endpoint Parameters section, all bundle endpoint clients need at least three pieces of information in order to be correctly configured: the foreign trust domain name, its bundle endpoint URL, and the endpoint profile.
+
+The bundle endpoint URL provides the address at which a bundle for the foreign trust domain can be found, and the profile tells clients which protocol should be used when calling it. A profile may require additional profile-specific parameters. Please see the relevant Endpoint Profiles subsection for more information about exactly how to connect to and authenticate a bundle endpoint.
+
+Once the connection has been successfully established, and a copy of the bundle has been received, it is stored along with the name of the trust domain it belongs to. The contents of the bundle (e.g. CA certificates, JWT signing keys, etc) can now be distributed for the purposes of validating SVIDs originating from the foreign trust domain.
+
+The exact nature and mechanism by which this distribution occurs is an implementation detail, and is out of scope for this document. For more information on how SPIFFE-aware workloads can receive bundle updates, please see the [SPIFFE Workload API][10] specification.
+
+### 6.2. Maintaining a Relationship
+SPIFFE bundle endpoint clients poll the bundle endpoint periodically for updates. When an update is detected, the stored bundle representing the endpoint's foreign trust domain is updated to match. The updated content is then distributed so that validators can add new keys and drop revoked keys as necessary. Again, the exact method(s) for distributing this update to validators is out of scope for this document.
```

Comment:
Per another comment in this PR, I think it should be: "...clients SHOULD poll..."